aptos-move/framework/aptos-framework/sources/transaction_fee.spec.move

```diff
@@ -133,4 +161,4 @@ spec aptos_framework::transaction_fee {
         aborts_if exists<AptosCoinCapabilities>(addr);
         ensures exists<AptosCoinCapabilities>(addr);
     }
-}
+}
```

Comment:
Add a new line after this line. Otherwise, the linter fails: https://github.com/aptos-labs/aptos-core/actions/runs/5206823455/jobs/9393770795?pr=8558#step:5:80